com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1301,6 +1306,8 @@ internal void ShutdownInternal()
             IsListening = false;
             m_ShuttingDown = false;
             m_StopProcessingMessages = false;
+
+            ClearClients();
```

Comment:
to answer your question, I'd like to ask a question here. wouldn't this change behavior such that connectedclients list(s) would be empty after stophost/client/server (aka shutdown) instead of contain stale data until we call Initialize() again?
I think that's the fix we're going for here with the JIRA ticket & this PR.
so, a changelog entry saying something like, "cleanup client lists on stophost/client/server" would be useful to know I think.

Reply:
ok. done.